src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER_BOUND = 101;
+    private final Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String color = String.valueOf(colorSupplier.getRandomColor());
```

Comment:
Don't use String.valueOf, when you fix the comment in ColorSupplier, your method will return String